src/rendering/vulkan/shaders/vk_shader.cpp

```diff
@@ -38,7 +38,7 @@ VkShaderManager::VkShaderManager(VulkanDevice *device) : device(device)
 			FString defines = defaultshaders[usershaders[i].shaderType].Defines + usershaders[i].defines;
 
 			VkShaderProgram prog;
-			prog.vert = LoadVertShader(name, mainvp, defaultshaders[i].Defines);
+			prog.vert = LoadVertShader(name, mainvp, defaultshaders[usershaders[i].shaderType].Defines);
```

Comment:
This is wrong, the argument should be just `defines`. User’s definitions will be ignored otherwise.

Reply:
But are they really needed for the vertex shader?